fuzz/init-fuzzer.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# https://stackoverflow.com/a/246128/3549270
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+cd "$SCRIPT_DIR"
+
+if ! cargo fuzz --help >&/dev/null; then
+  cargo install --git https://github.com/rust-fuzz/cargo-fuzz.git
+fi
```

Comment:
Should we ask users to install `cargo-fuzz` themselves?

Reply:
Potentially; however, cargo fuzz recommends installing via `cargo install cargo-fuzz`, but their release cycle is very slow and the main branch is normally stable. For having all potential features available, it's better to install via the git and I figured it would be easier to do so in a script.